src/formatDate.js

```diff
@@ -8,7 +8,29 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
The use of `pop()` on `fromFormat` and `toFormat` assumes that the last element is always the separator. Ensure that this is the intended design, as `pop()` modifies the original array, which might lead to unexpected behavior if the arrays are used elsewhere.